tests/serialize/runstate/dynamodb_state_store_test.py

```diff
@@ -115,221 +57,135 @@ def small_object():
 
 
 @pytest.fixture
-def large_object():
-    yield {
+def large_jobrun_state_object():
+    """Models a JobRun state object that is larger than the partition size."""
+    base_run = {
+        "job_run_id": "example_job.1",
+        "action_name": "large_step",
+        "state": "succeeded",
+        "original_command": "echo test",
+        "start_time": None,
+        "end_time": None,
+        "node_name": "example_node",
+        "exit_status": 0,
+        "attempts": [],
+        "retries_remaining": None,
+        "retries_delay": None,
+        "action_runner": {"status_path": "/tmp/status", "exec_path": "/usr/bin"},
+        "executor": "ssh",
+        "trigger_downstreams": None,
+        "triggered_by": None,
+        "on_upstream_rerun": None,
+        "trigger_timeout_timestamp": None,
+    }
+    return {
         "job_name": "example_job",
         "run_num": 1,
         "run_time": None,
         "time_zone": None,
         "node_name": "example_node",
-        "runs": [],
         "cleanup_run": None,
         "manual": False,
-        "large_data": [i for i in range(1_000_000)],
+        "runs": [base_run] * 5000,  # gives us a ~2000KB JobRun, ensuring at least a few partitions.
     }
 
 
-@pytest.mark.usefixtures("store", "small_object", "large_object")
+@pytest.mark.usefixtures("store", "job_state_object", "small_jobrun_state_object", "large_jobrun_state_object")
```

Comment:
just curious (in case you know): do we need pytest.mark.usefixtures here? seems like we actually refer to the fixtures in our tests and, outside of the `store` fixture, the other fixtures only return data